### ISSUE TITLE
Element link is missing required attribute href

### ISSUE THREAD

**Issue by dziku86** [validator/validator#1128]
URL being validated or code to reproduce error:

```HTML
<link rel="preload" as="image"  imagesrcset="wolf_400px.jpg 400w, wolf_800px.jpg 800w" imagesizes="50vw">
```
Allow `link` element to not have a `href` attr when `imagesrcset` attr is present.

Explanation:

https://html.spec.whatwg.org/dev/semantics.html#attr-link-imagesrcset

> Note how we omit the href attribute, as it would only be relevant for browsers that do not support imagesrcset, and in those cases it would likely cause the incorrect image to be preloaded.

Even though `imagesrcset` and `imagesizes` attrs are not allowed on element `link` at this point, as the error is stating, missing `href` attr shouldn't be an error IMHO.

**Comment by sideshowbarker** [comment]
Yeah the root problem here is that the checker doesn’t yet know anything about `imagesrcset` and `imagesizes`.

So the fix is that I need to add support for those.

**Comment by sideshowbarker** [comment]
I see now this was also reported in https://github.com/validator/validator/issues/826. I’ll make time next week to get this in.

**Comment by johnkw** [comment]
Isn't this just a duplicate of bug #826 ? Seems like this one should be closed in favor of that one.

**Comment by dziku86** [comment]
> Isn't this just a duplicate of bug #826 ? Seems like this one should be closed in favor of that one.

Ok. I'm closing this.
